generator/template/default/_browserslistrc

```diff
@@ -1,3 +0,0 @@
-> 1%
-last 2 versions
-not dead
```

Comment:
Please explain why are we removing this?

Reply:
This is already defined in package.json file generated by vue-cli.
![image](https://user-images.githubusercontent.com/2633898/100094434-4f293600-2e59-11eb-9c97-736f08227be2.png)

Reply:
Hmm I think this is a matter of choice done during initalization phase - user can choose does he wish to keep these settings (along with others) in package.json or in dedicated files.

Reply:
Yes, user can choose where to keep settings, but if you use default vue-3 config there is no prompt which ask you where you want save those settings.